modules/trustx2BidAdapter.js

```diff
@@ -0,0 +1,494 @@
+import {
+  logInfo,
+  logError,
+  logMessage,
+  deepAccess,
+  deepSetValue,
+  mergeDeep,
+  ajax
+} from '../src/utils.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER, VIDEO} from '../src/mediaTypes.js';
+import {Renderer} from '../src/Renderer.js';
+import {ortbConverter} from '../libraries/ortbConverter/converter.js';
+import {config} from '../src/config.js';
+import {getStorageManager} from '../src/storageManager.js';
+
+/**
+ * @typedef {import('../src/adapters/bidderFactory.js').BidRequest} BidRequest
+ * @typedef {import('../src/adapters/bidderFactory.js').Bid} Bid
+ * @typedef {import('../src/adapters/bidderFactory.js').ServerResponse} ServerResponse
+ * @typedef {import('../src/adapters/bidderFactory.js').SyncOptions} SyncOptions
+ */
+
+const BIDDER_CODE = 'trustx2';
```

Comment:
i think this is a large mistake; you should endeavor to keep your installed base. See the newspass transition from ozone to aditude backend as an example